libraries/heft-config-file/src/ConfigurationFileBase.ts

```diff
@@ -109,6 +186,10 @@ export interface IJsonPathMetadataResolverOptions<TConfigurationFile> {
    * The configuration file the property was obtained from.
    */
   configurationFile: Partial<TConfigurationFile>;
+  /**
+   * If this is a project configuration file, the root folder of the project.
```

Comment:
Mention that this will be resolved if needed and not provided.

Reply:
This particular object is the data provided _by the loader_ so the description as written is accurate.